packages/api/amplify_api/test/web_socket/web_socket_bloc_test.dart

```diff
@@ -179,9 +182,9 @@ void main() {
         await dataCompleter.future;
         dataCompleter = Completer<String>();
 
-        fakePlatform.controller.sink.add(ConnectivityResult.none);
-
-        fakePlatform.controller.sink.add(ConnectivityResult.mobile);
+        mockNetworkStreamController
+          ..add(ConnectivityStatus.disconnected)
+          ..add(ConnectivityStatus.connected);
```

Comment:
I like this more than the previous mock setup, good job! Is it possible to add the actual network result to the stream and let the .map() transform it into a `ConnectivityStatus`? Doesn't look like we have any tests around that branch yet.